site/content/contribute/good-decisions/fixing-pr-mistakes.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Merged PR Scenario"
```

Comment:
Let's reword this new page title to describe the reader's circumstance. What scenario would bring a user to this page to learn more?

Reply:
Reworded title.